Mathlib/Analysis/SpecialFunctions/Exp.lean

```diff
@@ -136,23 +136,23 @@ variable {α : Type*}
 
 open Real
 
-theorem Filter.Tendsto.exp {l : Filter α} {f : α → ℝ} {z : ℝ} (hf : Tendsto f l (𝓝 z)) :
+theorem Filter.Tendsto.rexp {l : Filter α} {f : α → ℝ} {z : ℝ} (hf : Tendsto f l (𝓝 z)) :
     Tendsto (fun x => exp (f x)) l (𝓝 (exp z)) :=
   (continuous_exp.tendsto _).comp hf
-#align filter.tendsto.exp Filter.Tendsto.exp
+#align filter.tendsto.exp Filter.Tendsto.rexp
 
 variable [TopologicalSpace α] {f : α → ℝ} {s : Set α} {x : α}
 
 nonrec
 theorem ContinuousWithinAt.exp (h : ContinuousWithinAt f s x) :
```

Comment:
I think we want to do the same to these, but I wouldn't object if you wanted to do that in a follow-up

Reply:
That's perhaps mission creep for this PR. But I agree that it should be done, and I can have a look.